SharedProjects/BabylonExport.Entities/BabylonMaterial.cs

```diff
@@ -23,6 +23,8 @@ public partial class BabylonMaterial
         [DataMember]
         public int alphaMode { get; set; }
 
+
```

Comment:
darn it.

Reply:
corrected via https://github.com/BabylonJS/Exporters/commit/8be365a8ae1c3ae39d0713fa21e2c9b6b4c6fdd9